### ISSUE TITLE
Need to seperate training set, test set when using supervised topic modeling

### ISSUE THREAD

**Issue by dlrudgnsdlek** [MaartenGr/BERTopic#1305]
Hi author, thank you for offering this wonderfful tool. 

I have 200 labeled data, and 100000 unlabeled data. I want to classify this unlabeld paragraphs by using supervised topic modeling. In the tutorial page, you used fin_transform when doing supervised topic modeling. I think when I use fit_transform, there is no need to split data into train set, test set in labeled data. Is that right? Then how can i evaluate the trained model?

**Comment by MaartenGr** [comment]
It depends, if you are using a fully supervised approach, then you are essentially doing classical supervised learning which typically involves train/test/dev/etc. splits. Moreover, evaluation then not only depends on the supervised approach itself but also on the quality of the resulting topics.

**Comment by MaartenGr** [comment]
Closing this due to inactivity. Let me know if you want to re-open the issue!
